xamarin/Xamarin.Forms
# [Core] Shapes not rendering changing Shell tabs

### Description of Change ###

Shapes not rendering changing Shell tabs. 

### Issues Resolved ### 

- fixes #11924 
- fixes #11656
- fixes #12844

### API Changes ###
 
 None

### Platforms Affected ### 

- Core/XAML (all platforms)

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

![fix11924-ios](https://user-images.githubusercontent.com/6755973/91295957-229f4880-e79c-11ea-9a6e-2905791b1ea7.gif)
![fix11924-droid](https://user-images.githubusercontent.com/6755973/91295958-23d07580-e79c-11ea-9af5-b285a6064506.gif)


### Testing Procedure ###
Launch Core Gallery and navigate to the issue 11924. Navigate to the second Tab and then navigate back to the first tab, and verify if the Ellipse is rendering.

### PR Checklist ###
